tests/L0/run_fused_layer_norm/test_fused_layer_norm.py

```diff
@@ -269,5 +269,41 @@ def test_autocast_fused_rms_norm(self, dtype, elementwise_affine):
 
 instantiate_device_type_tests(TestFusedLayerNorm, globals(), only_for=("cuda",))
 
+def _verify_export(fused, fused_x):
+    # check that export() is working
+    onnx_str = torch.onnx.export_to_pretty_string(fused, (fused_x,),
+                                                  input_names=['x_in'],
+    )
+    assert('x_in' in onnx_str)
+    assert('ReduceMean' in onnx_str)
+
+        
+def test_export_fused_rms_norm():
+    batch_size = 16
+    normalized_shape = [32, 16]
+    fused = FusedRMSNorm(
+        normalized_shape=normalized_shape, elementwise_affine=True
+    ).cuda()
+    fused_m = MixedFusedRMSNorm(
+        normalized_shape=normalized_shape, elementwise_affine=True
+    ).cuda()
+    native_x, fused_x = _prep_inputs(batch_size, normalized_shape, torch.float32)
+    _verify_export(fused, fused_x)
+    _verify_export(fused_m, fused_x)
+    
+
+def test_export_fused_layer_norm():
+    batch_size = 16
+    normalized_shape = [32, 16]
+    fused = FusedLayerNorm(
+        normalized_shape=normalized_shape, elementwise_affine=True
+    ).cuda()
+    fused_m = MixedFusedLayerNorm(
+        normalized_shape=normalized_shape, elementwise_affine=True
+    ).cuda()
+    native_x, fused_x = _prep_inputs(batch_size, normalized_shape, torch.float32)
+    _verify_export(fused, fused_x)
+    _verify_export(fused_m, fused_x)
```

Comment:
Could you implement these as [`unittest.TestCase`](https://docs.python.org/3/library/unittest.html#unittest.TestCase)'s methods so that an environment w/o pytest can also run?

Reply:
Done.